src/_cffi_src/openssl/rand.py

```diff
@@ -9,7 +9,6 @@
 """
 
 TYPES = """
-static const long Cryptography_HAS_EGD;
```

Comment:
Sure, why not. At some point it'd be good to remove it entirely but the two lines aren't going to be a maintenance burden in any way...